src/elements/ops.rs

```diff
@@ -115,7 +115,7 @@ pub enum Instruction {
 	End,
 	Br(u32),
 	BrIf(u32),
-	BrTable(Box<[u32]>, u32),
+	BrTable(Box<BrTableData>),
```

Comment:
I wonder why this change was made?

Reply:
There was a test asserting that the size of `Instruction` was less than 24 bytes and the new instructions caused this limit to be blown, so this, the largest variant, was changed to store data out-of-line to keep the size of `Instruction` under or at 24 bytes

Reply:
Fair enough!